packages/scene-composer/src/utils/colorUtils.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { isValidHexCode } from './colorUtils';
```

Comment:
nit: Do we have our own way of doing this? ThreeJS has this this built in: https://threejs.org/docs/#api/en/math/Color

We shouldn't need to have tests for color conversion or validity, we should rely on a library that handles this for us.